src/tests/ftest/server/metadata.py

```diff
@@ -190,101 +187,87 @@ def test_metadata_addremove(self):
-                return "FAIL"
-        self.d_log.debug("IOR {0} Threads Finished -----".format(operation))
-        return "PASS"
-
     @avocado.fail_on(DaosApiError)
     def test_metadata_server_restart(self):
-        """
-        Test ID: DAOS-1512
-        Test Description: This test will verify 2000 IOR small size container
-                          after server restart. Test will write IOR in 5
-                          different threads for faster execution time. Each
-                          thread will create 400 (8bytes) containers to the
-                          same pool. Restart the servers, read IOR container
-                          file written previously and validate data integrity
-                          by using IOR option "-R -G 1".
+        """JIRA ID: DAOS-1512.
+
+        Test Description:
+            This test will verify 2000 IOR small size container after server
+            restart. Test will write IOR in 5 different threads for faster
+            execution time. Each thread will create 400 (8bytes) containers to
+            the same pool. Restart the servers, read IOR container file written
+            previously and validate data integrity by using IOR option
+            "-R -G 1".
+
+        Use Cases:
+            ?
+
         :avocado: tags=metadata,metadata_ior,nvme,small
         """
-        self.pool_connect = False
-        files_per_thread = 400
-        total_ior_threads = 5
-        threads = []
-        ior_args = {}
-
-        createsvc = self.params.get("svcn", '/run/pool/createsvc/')
-        svc_list = ""
-        for i in range(createsvc):
-            svc_list += str(int(self.pool.svc.rl_ranks[i])) + ":"
-        svc_list = svc_list[:-1]
-
-        ior_args['client_hostfile_servers'] = self.hostfile_clients
-        ior_args['pool_uuid'] = self.pool.get_uuid_str()
-        ior_args['svc_list'] = svc_list
-        ior_args['basepath'] = self.basepath
-        ior_args['server_group'] = self.server_group
-        ior_args['tmp_dir'] = self.workdir
-        ior_args['iorwriteflags'] = self.params.get("F",
-                                                    '/run/ior/iorwriteflags/')
-        ior_args['iorreadflags'] = self.params.get("F",
-                                                   '/run/ior/iorreadflags/')
-        ior_args['iteration'] = self.params.get("iter", '/run/ior/iteration/')
-        ior_args['stripe_size'] = self.params.get("s", '/run/ior/stripesize/*')
-        ior_args['stripe_count'] = self.params.get("c", '/run/ior/stripecount/')
-        ior_args['async_io'] = self.params.get("a", '/run/ior/asyncio/')
-        ior_args['object_class'] = self.params.get("o", '/run/ior/objectclass/')
-        ior_args['slots'] = self.params.get("slots", '/run/ior/clientslots/*')
-
-        ior_args['files_per_thread'] = files_per_thread
+        # files_per_thread = 400
+        files_per_thread = 2
+        # total_ior_threads = 5
```

Comment:
Are these meant to be commented out or are they "quick testing" left-overs?

Reply:
Quick testing - thanks for catching this.